tools/dm/dm-configuration-file-overview.md

```diff
@@ -10,7 +10,7 @@ This document gives an overview of configuration files of DM (Data Migration).
 
 ## DM process configuration files
 
-- `inventory.ini`: The configuration file of deploying DM using DM-Ansible. You need to edit it based on your machine topology. For details, see [Edit the `inventory.ini` file to orchestrate the DM cluster](/tools/dm/deployment.md#step-7-edit-the-inventoryini-file-to-orchestrate-the-dm-cluster).
+- `inventory.ini`: The configuration file of deploying DM using DM-Ansible. You need to edit it based on your machine topology. For details, see [Edit the `inventory.ini` file to orchestrate the DM cluster](/tools/dm/deployment.md#step-7-edit-the-inventory-ini-file-to-orchestrate-the-dm-cluster).
```

Comment:
Please remove the ".md" from and add back slash to the link  '/tools/dm/deployment.md#step-7-edit-the-inventory-ini-file-to-orchestrate-the-dm-cluster'